packages/manager/apps/hycu/src/pages/dashboard/general-information/BillingInformations/BillingInformationsTile.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  DashboardTile,
+  DateFormat,
+  Description,
+  Renew,
+  useFormattedDate,
+  useServiceDetails,
+} from '@ovh-ux/manager-react-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components';
+import {
+  OsdsIcon,
+  OsdsLink,
+  OsdsSkeleton,
+} from '@ovhcloud/ods-components/react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigationGetUrl } from '@/hooks/shell/useNavigationGetUrl';
+
+const BillingInformationsTile = ({ serviceName }: { serviceName: string }) => {
+  const { t } = useTranslation('hycu/dashboard');
+
+  const { data: serviceDetails, isLoading } = useServiceDetails({
+    resourceName: serviceName,
+  });
+
+  const renewDate = useFormattedDate({
+    dateString: (serviceDetails?.data.billing.renew as Renew)?.current.nextDate,
+    format: DateFormat.display,
+  });
+
+  const creationDate = useFormattedDate({
+    dateString: serviceDetails?.data.billing.lifecycle.current.creationDate,
+    format: DateFormat.display,
+  });
+
+  const {
+    data: contactUrl,
+    isLoading: isContactUrlLoading,
+  } = useNavigationGetUrl([
+    'dedicated',
+    '#/contacts/services',
+    { serviceName },
+  ]);
+
+  return (
+    <DashboardTile
+      title={t('hycu_dashboard_subscription_title')}
+      items={[
+        {
+          id: 'renew',
+          label: t('hycu_dashboard_label_renew'),
+          value: isLoading ? (
+            <OsdsSkeleton />
+          ) : (
+            <OsdsLink color={ODS_THEME_COLOR_INTENT.info}>{renewDate}</OsdsLink>
+          ),
+        },
+        {
+          id: 'contact',
+          label: t('hycu_dashboard_field_label_contacts'),
+          value: (
+            <div className="flex flex-col gap-4">
+              <div>
+                {serviceDetails?.data.customer.contacts.map((contact) => {
+                  return (
```

Comment:
**loading**: maybe you should add loading management here as well
**nitpick**: could use implicit return